### PR TITLE
Show/Hide Finished Chores

### PR DESCRIPTION
**Pull request by otaviocc**
and match navigation bar style with other parts of the app.

Simulator Screen Shot - iPhone 11 - 2020-06-06 at 17 24 31

### CODE REVIEW

File: ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift [hunk 1]
```diff
@@ -260,8 +260,8 @@ public class UserCollection: ObservableObject {
         save()
     }
 
-    public func deleteChore(at index: Int) {
-        chores.remove(at: index)
+    public func deleteChore(_ chore: Chore) {
+        chores.removeAll { $0.id == chore.id }
```
Comment (renaudjenny) [inline ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift:264]: `Chore` is `Equatable` so you can directly compare them
```suggestion
        chores.removeAll { $0 == chore }
```

Comment (otaviocc) [reply ACHNBrowserUI/ACHNBrowserUI/packages/Backend/Sources/Backend/environments/UserCollection.swift]: Done. Thanks @renaudjenny!

File: ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayChoresSection.swift [hunk 2]
```diff
@@ -24,7 +24,7 @@ struct TodayChoresSection: View {
 
     var body: some View {
         Section(header: SectionHeaderView(text: "Chores", icon: "checkmark.seal.fill")) {
-            NavigationLink(destination: ChoreListView()) {
+            NavigationLink(destination: LazyView(ChoreListView())) {
```
Comment (TheVaan) [inline ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayChoresSection.swift:27]: As of Xcode 11.4.1, iOS 13.4.1, Apple fixed NavigationLink destination loading and views are now loaded lazily. So this shouldn’t be necessary?

Comment (otaviocc) [reply ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayChoresSection.swift:27]: I tried without, but I still face the problem.

**Without Lazy**: When hiding Finished Task and adding new chores, it resets and shows all the chores.

wo

**With Lazy**: When hiding Finished Tasks and adding new chores, it behaves as expected.

w

Comment (TheVaan) [reply ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayChoresSection.swift:27]: Interesting 🤔

**Pull request merged**